capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -12,6 +12,31 @@ const DOWN_KEY = 'ArrowDown';
 const ENTER_KEY = 'Enter';
 let selectedSuggestionPosition = -1;
 
+/**
+ * Gets suggestions to display to the users
+ */
+function suggestionInput() {
+  const val = SEARCH_INPUT.value.trim();
+  if (!val) {
+    return;
+  }
+  const fetchStr = '/search?query=' + val;
+  fetch(fetchStr).then((response) => response.json()).then((suggestions) => {
+    getsAllProposedSuggestions(suggestions, val);
+  });
+}
+
+/**
+ * Sends final college list to the dashboard
+ */
+function sendCollegeInformationToDashboard() {
+  const listItems = document.getElementById('suggestions').children;
+  const listArray = Array.from(listItems);
+  const idArray = Array.from(listItems).map((l) => l.id).join(',');
```

Comment:
nit - consider renaming this variable since it's no longer an array.

Reply:
done!